core/components/organisms/verticalNav/MenuItem.tsx

```diff
@@ -127,20 +172,22 @@ export const MenuItem = (props: MenuItemProps) => {
   ) : (
     // TODO(a11y)
     // eslint-disable-next-line
-    <Link componentType="a" className={ItemClass} {...baseProps}>
-      <div className="d-flex align-items-center overflow-hidden">
-        {menu.icon && (
-          <Icon
-            data-test="DesignSystem-VerticalNav--Icon"
-            className={expanded ? 'mr-4' : ''}
-            name={menu.icon}
-            appearance={getIconAppearance(isActive, menu.disabled)}
-          />
-        )}
-        {expanded && <MenuLabel label={menu.label} disabled={menu.disabled} isActive={isActive} />}
-      </div>
-      {expanded && renderSubMenu()}
-    </Link>
+    <div id="vertical-nav-menu">
+      <Link componentType="a" className={ItemClass} {...baseProps}>
```

Comment:
Please check if we can directly give `data-id` to `Link` component and then compute the height

Reply:
By calculating from Link we get the wrong height and the animation gets a little abrupt at last